src/context.tsx

```diff
@@ -112,6 +113,11 @@ const reducer = (state: State, action: Record<string, any>): State => {
       pageState.estimateGradePrevalences = estimateGradePrevalences;
       return newState;
     };
+    case "UPDATE_COUNTRIES_JSON":
```

Comment:
Hmm how often do we expect the WHO to update their map tiles. If it's not likely to be frequent, maybe we can just store the countries JSON in a static file and update it on a schedule? That would help us achieve some minor performance wins

Reply:
It's a good point, might not be necessary just yet though if we don't have to spend anything to do this

Reply:
I think it's the WHOs expectation that we are always pulling data from their endpoints whenever possible. We shouldn't store anything static, so we are guaranteed to have the most up-to-date geo data

Reply:
Okay cool that's fair